js/src/test/binary_serialization.ts

```diff
@@ -1,46 +1,46 @@
 import { array_or_json } from "../serialize";
 import { expect } from 'chai';
-// import {DummyManager} from './dummy-manager';
-// import bqplot = require('..');
-// import {Figure} from '../src/Figure.js';
-// import {FigureModel} from '../src/FigureModel.js';
-// import {create_model, create_model_bqplot, create_view, create_figure_scatter} from './widget-utils'
-// import * as d3 from 'd3';
 
 
 describe("binary serialization >", () => {
     beforeEach(async function() {
     });
 
-    it("arrays", async function() {
-        let x_wire = {dtype: 'float32', value: new DataView((new Float32Array([0,0.5,1])).buffer)}
-        let x_ar = array_or_json.deserialize(x_wire, null);
-        let x =  Array.prototype.slice.call(x_ar);
+    it("deserialize/serialize number arrays", async function() {
+        const x_wire = {dtype: 'float32', value: new DataView((new Float32Array([0,0.5,1])).buffer)}
+        const x_ar = array_or_json.deserialize(x_wire, null);
+        const x =  Array.prototype.slice.call(x_ar);
         expect(x).to.deep.equal([0, 0.5, 1.0])
-        let x_wire_bytes = Array.prototype.slice.call(new Uint8Array(x_wire.value.buffer))
+        const x_wire_bytes = Array.prototype.slice.call(new Uint8Array(x_wire.value.buffer))
 
-        let x_ar2 = new Float32Array([0, 0.5, 1.])
-        let x_wire2 = array_or_json.serialize(x_ar2, null);
-        let x_wire2_bytes = Array.prototype.slice.call(new Uint8Array(x_wire2.value.buffer))
+        const x_ar2 = new Float32Array([0, 0.5, 1.])
+        const x_wire2 = array_or_json.serialize(x_ar2, null);
+        const x_wire2_bytes = Array.prototype.slice.call(new Uint8Array(x_wire2.value.buffer))
         expect(x_wire_bytes).to.deep.equal(x_wire2_bytes)
     });
 
-    it("date arrays", async function() {
+    it("deserialize/serialize date arrays", async function() {
         // 2005-02-25 and 2 days after:
-        let x_wire = {dtype: 'float64', value: new DataView((new Float64Array([1109289600000, 1109376000000, 1109462400000])).buffer), type: 'date'}
-        let x_ar = array_or_json.deserialize(x_wire, null);
+        const x_wire = {dtype: 'float64', value: new DataView((new Float64Array([1109289600000, 1109376000000, 1109462400000])).buffer), type: 'date'}
+        const x_ar = array_or_json.deserialize(x_wire, null);
         expect(x_ar.type).to.equal('date')
-        let x =  Array.prototype.slice.call(x_ar);
+        const x =  Array.prototype.slice.call(x_ar);
```

Comment:
FYI: I used slice because I didn't know any better, but I now use a cleaner solution like `[...x_ar]`. You don't have to change it now.